auth/LICENSE

```diff
@@ -0,0 +1,214 @@
+THE ACCOMPANYING PROGRAM IS PROVIDED UNDER THE TERMS OF THIS ECLIPSE PUBLIC
```

Comment:
This file should just be deleted.  If kept, it needs to be the Apache 2.0 text, **not** EPL.